### DISCUSSION TITLE
Display goes black but audio still plays when entering fullscreen

### DISCUSSION THREAD

**Discussion by TrainingArc** [FreeTubeApp/FreeTube#6122] [Q&A]
I've encountered this a few times even after uninstalling and reinstalling FreeTube, even with RevoUninstaller. When entering fullscreeen on a video, the entire display goes black after a split second and will stay black so long as the window is selected. I have also encountered an issue where sometimes the app will launch and make the display black which is what prompted me to uninstall and reinstall the program in the first place

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @TrainingArc is this still happening to you?

**Comment by alejandrod19-ship-it** [comment]
Happening to me. Windows 11 V0.25.1
